Firmware/MotorControl/encoder.cpp

```diff
@@ -522,6 +522,11 @@ bool Encoder::update() {
         snap_to_zero_vel = true;
     }
 
+    //expose pos/vel estimates in radians for Controller
```

Comment:
IMO it shoudl be semantically the case that the "main output" of the Encoder is results in radians, and the count based variables are kind of an internal implementation detail (still ok to expose it of course).

Reply:
Fixed in 7ce896c